crates/core/migrations/2022-01-30-195925_create_storev2_tables/up.sql

```diff
@@ -0,0 +1,45 @@
+create table storefrontsv2_configs (
+  address                         varchar(48)    PRIMARY KEY,
+  settings_uri                    varchar(200)
+);
+
+create table storefrontsv2(
+  store_address                     varchar(48)    PRIMARY KEY,
+  public                            boolean,
+  auction_program                   varchar(48),
+  token_vault_program               varchar(48) ,
+  token_metadata_program            varchar(48),
+  token_program                     varchar(48),
+  store_config_pda                  varchar(48)
+);
+
+
+create table whitelisted_creators (
+  address                         varchar(48),
+  creator_address                 varchar(48),
```

Comment:
Lets add index on creator_address.

Reply:
its added for address and creator_address, do you want only for creator_address

Reply:
unique index for creator_address it will be the most likely where clause.